deployment/confStorage/conf_storage_util.py

```diff
@@ -60,8 +60,13 @@ def create_path(path):
 
 def read_file_from_path(file_path):
 
-    with open(file_path, "r") as fin:
```

Comment:
Don't change the uncorrelated module's function. This change will have potential issue. Please revert it.

Reply:
That's what we need, we want to tolerant non existing file.

Reply:
Got you, it's  not the right place